modules/virt-hot-plugging-a-virtual-disk.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk_{context}"]
+= Hot-plugging a virtual disk
+
+Hot-plug virtual disks that you want to add to or remove from a virtual machine instance while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+* You must have at least one data volume or persistent volume claims (PVC) available for hot-plugging.
+
+To hot-plug a virtual disk using the CLI, use the `addvolume` command with `virtctl`.
+
+.Procedure
+
+. Run the following command:
+
+----
+$ virtctl addvolume <virtual-machine|virtual-machine-instance> --volume-name=<datavolume-or-PVC-to-hot-plug> [--persist] [--serial=<label-name>]
+----
+
+* The `addvolume` command creates an attachment pod to attach the hot-plugged disk.
+* The optional `--persist` flag adds the hot-plugged disk to the virtual machine specification as a permanently mounted virtual disk. After running the command with this option, the virtual disk is no longer able to be hot-unplugged. In other words, the virtual disk can no longer be added to or removed from the virtual machine while the virtual machine is running. The `--persist` flag applies to virtual machines, not virtual machine interfaces.
+* The optional `--serial` flag allows you to add an alphanumeric string label of your choice, which helps you to identify the hot-plugged disk. If you do not specify this option, the label defaults to the name of the hot-plugged datavolume or PVC.
```

Comment:
Helps you identify the disk INSIDE the guest, which is the critical detail why we care about the serial.

Reply:
Changed to: 

_The optional `--serial` flag allows you to add an alphanumeric string label of your choice, which helps you to identify the hot-plugged disk in the guest VMI. If you do not specify this option, the label defaults to the name of the hot-plugged datavolume or PVC._

Reply:
in the guest VM, no VMI.

Reply:
Corrected